README.md

```diff
@@ -115,11 +116,21 @@ Note that even if enry's CLI is compatible with linguist's, its main point is th
 
 
 Development
------------
+------------
+
+*enry* re-uses parts of original [linguist](https://github.com/github/linguist)  to generate internal data structures. In order to update to latest upstream and generate the necessary code you must run:
+
+    go generate
+
+We update enry due to changes  in linguist's master branch related to the following files:
+* [languages.yml](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml)
+* [heuristics.rb](https://github.com/github/linguist/blob/master/lib/linguist/heuristics.rb)
+* [vendor.yml](https://github.com/github/linguist/blob/master/lib/linguist/vendor.yml)
+* [documentation.yml](https://github.com/github/linguist/blob/master/lib/linguist/documentation.yml)
 
-*enry* re-uses parts of original [linguist](https://github.com/github/linguist) especially data in `languages.yml` to generate internal data structures. In oreder to update to latest upstream run
+For the moment we don't have any procedure established to detect changes in the linguist project automatically and regenerate the code. So we are updating the generated code when is needed  without any specific criteria while we are going ahead in the project development.
```

Comment:
* s/generated code when is needed  without/generated code as needed, without/
* The last part (_while we are going...) not specially informative, could be removed